enterprise/server/scheduling/priority_task_scheduler/priority_task_scheduler.go

```diff
@@ -294,6 +295,7 @@ func (q *PriorityTaskScheduler) EnqueueTaskReservation(ctx context.Context, req
 }
 
 func (q *PriorityTaskScheduler) propagateExecutionTaskValuesToContext(ctx context.Context, execTask *repb.ExecutionTask) context.Context {
+	ctx = usageutil.InternalUsageContext(ctx)
```

Comment:
I think a comment would be useful here (and below as well, about what this does and why you're calling it?)

Reply:
Done - also renamed `InternalUsageContext` since the "Internal" part might be confusing (e.g. BYOX would actually be external-executor). wdyt?